lib/sycamore/sycamore/transforms/extract_table_properties.py

```diff
@@ -52,42 +52,48 @@ def extract_parent_json(input_string: str) -> str:
 
     @staticmethod
     @timetrace("ExtrKeyVal")
-    def extract_table_properties(parent: Document, property_name: str, llm: LLM) -> Document:
+    def extract_table_properties(
+        parent: Document, property_name: str, llm: LLM, prompt_find_table: str = "", prompt_LLM: str = ""
+    ) -> Document:
         """
         This Method is used to extract key value pair from table using LLM and
         populate it as property of that element.
         """
-        prompt = """
-        You are given a text string where columns are separated by comma representing either a single column, 
-        or multi-column table each new line is a new row.
-        Instructions:
-        1. Parse the table and make decision if key, value pair information can be extracted from it.
-        2. if the table contains multiple cell value corresponding to one key, the key, value pair for such table 
-        cant be extracted.
-        3. return True if table cant be parsed as key value pair.
-        4. return only True or False nothing should be added in the response.
-        """
-        query_agent = LLMTextQueryAgent(prompt=prompt, llm=llm, output_property="keyValueTable", element_type="table")
+        if prompt_find_table == "":
+            prompt_find_table = """
+            You are given a text string where columns are separated by comma representing either a single column, 
+            or multi-column table each new line is a new row.
+            Instructions:
+            1. Parse the table and make decision if key, value pair information can be extracted from it.
+            2. if the table contains multiple cell value corresponding to one key, the key, value pair for such table 
+            cant be extracted.
+            3. return True if table cant be parsed as key value pair.
+            4. return only True or False nothing should be added in the response.
+            """
```

Comment:
Can these default prompts be moved to the file with all the default prompts? I think it's like sycamore/llms/prompts.py or something.